packages/core/src/indexing-store/utils/sort.ts

```diff
@@ -32,12 +38,17 @@ export function buildOrderByConditions({
       `Invalid sort. Column does not exist. Got '${columnName}', expected one of [${Object.keys(
         table,
       )
-        .filter((key) => isBaseColumn(table[key]) || isEnumColumn(table[key]))
+        .filter(
+          (columnName) =>
+            isScalarColumn(table[columnName]) ||
+            isReferenceColumn(table[columnName]) ||
```

Comment:
So now, a reference column is not a scalar column?

Reply:
correct